Druid/DRCogitCanonicaliser.class.st

```diff
@@ -10,6 +10,41 @@ DRCogitCanonicaliser >> applyTo: cfg [
 	cfg instructions copy do: [ :e | e acceptVisitor: self ]
 ]
 
+{ #category : #visiting }
+DRCogitCanonicaliser >> putConstantsTogetherInDoubleInstructionIfPossible: aDRBinaryInstruction [
+	"If it is a pattern such as res := ((var op const1) op const2)"
```

Comment:
Now this method do more stuff.

I propose this comment:

```suggestion
DRCogitCanonicaliser >> putConstantsTogetherInDoubleInstructionIfPossible: aDRBinaryInstruction [
	"If it is a pattern such as ((var op const) op X)
	It is replaced by:
		((X op const) op var) -> if X is constant
		((var op X) op const) -> otherwise
	"
```

And maybe change the name to `sortOperandsWithContant...` or something like that.